### ISSUE TITLE
Problem with SSL/TLS

### ISSUE THREAD

**Issue by ruslanmedia** [yongkangchen/remote-sync#302]
Hi! I have a problem with connecting FTP
`[10:20:41 AM] Connecting: sgcc.uz@sgcc.uz:21
Error: SSL/TLS required on the control channel`

This server works with other FTP clients. How can I solve this problem?

**Comment by enoversum** [comment]
@ruslanmedia @elistone Having the exact same problem, FTP and SFTP work fine, FTPS not :cry: .

**Comment by elistone** [comment]
@ruslanmedia @enoversum Possible fix? #298 has not been checked by anyone yet though.

**Comment by enoversum** [comment]
@elistone Should we manually edit your plugin’s files to check it out?

**Comment by elistone** [comment]
Sure, if it works add a thumbs up/comment to the PR

**Comment by enoversum** [comment]
@elistone Unfortunately, it does not seem to work. I edited the FtpTransport.coffee file as described, added the option manually to the .remote-sync.json file, but upload is not working anymore. As soon as I remove the changes, FTP works again, but throws the known error on FTPS.

**Comment by elistone** [comment]
@enoversum so what you are saying is PR #298 from your quick testing seems to not really do anything other than break stuff?

**Comment by enoversum** [comment]
@elistone Yes, I changed the two lines on the said file and wasn't able to upload/download anymore, plus I didn't see any option for secure (checkbox?) connection on the create connection screen, and it made no difference if I added the secure: true line to the connection settings .json file or not (so it already broke by just editing the .coffee file).

**Comment by enoversum** [comment]
@elistone Any news on this matter? Just stumbled over another FTPS site I need to upload to 😉 .

**Comment by rfiorentino** [comment]
I'm also needing FTPS (implicit TLS) so I can say goodbye to Dreamweaver. Any hope for this?
